test/Microsoft.AspNet.Server.KestrelTests/EngineTests.cs

```diff
@@ -54,6 +55,14 @@ ILibraryManager LibraryManager
             }
         }
 
+        ILoggerFactory LoggerFactory
+        {
+            get
+            {
+               return new LoggerFactory();
```

Comment:
A property that always returns a new instance seems like an anti-pattern. Could the individual unit tests just create their own logger factories inline?